### PR TITLE
Cleaned up DRY violations in widget file

### PR DESCRIPTION
**Pull request by sw00d**
Give her a look baby

### CODE REVIEW

File: ThemeConfig.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import Foundation
```
Comment (gibsonbailey) [inline ThemeConfig.swift:8]: Were there any project file changes that arose in your `git diff` when you added this file?

Comment (sw00d) [reply ThemeConfig.swift:8]: Kinda confused on what you mean. It was in the diff because it was a new file.

Comment (gibsonbailey) [reply ThemeConfig.swift:8]: XCode has some metadata that it tracks about files, not like other IDEs I've used. Files have to be "Added" to the project with build targets specified on each one. They don't show up in XCode when they simply exist in the same directory as a project. 

This is why you might see broken references and files not showing up in the file tree.

That info is kept in the project files, so it has to be committed as well.

Comment (sw00d) [reply ThemeConfig.swift:8]: Lovely. I saw that file and didn't commit it. And then did a hard reset so lost the changes. I can just delete this file and commit it again and maybe that'll fix it

Comment (gibsonbailey) [reply ThemeConfig.swift:8]: You can go into XCode at File > Add File to make it show up. Should see some project files change when you do that. Commit those.

Comment (gibsonbailey) [reply ThemeConfig.swift:8]: Me gusta este archivo de color. Muchas gracias. Me encanta su trabajo.

Comment (sw00d) [reply ThemeConfig.swift:8]: Gracias. Acabo de cometer el archivo que pediste 📝

Comment (gibsonbailey) [reply ThemeConfig.swift:8]: Do you see that there are two `ThemeConfig.swift` files now?
Eres un tonto.

Comment (sw00d) [reply ThemeConfig.swift:8]: Cabron, que me importa. Ya voy a dormir. Lo haré mañana

Comment (gibsonbailey) [reply ThemeConfig.swift:8]: Muy bien. Puedes. Hasta mañana.

Comment (sw00d) [reply ThemeConfig.swift:8]: Ya como esta?

Comment (gibsonbailey) [reply ThemeConfig.swift:8]: Pienso que es bueno.

**Review by gibsonbailey — APPROVED** [review]

**Pull request merged**